core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -773,9 +761,9 @@ class DynamicBrokerReconfigurationTest extends QuorumTestHarness with SaslSetup
     consumer.commitSync()
   }
 
-  @ParameterizedTest(name = "{displayName}.groupProtocol={0}")
-  @CsvSource(Array("classic, consumer"))
-  def testThreadPoolResize(groupProtocol: String): Unit = {
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testThreadPoolResize(quorum: String, groupProtocol: String): Unit = {
```

Comment:
This gets flaky and I open https://issues.apache.org/jira/browse/KAFKA-18123 to fix it.